src/types/api.js

```diff
@@ -30,6 +36,12 @@
  * @prop {string} logo
  */
 
+/**
+ * @typedef GroupScopes
+ * @prop {boolean} enforced
+ * @prop {string[]} uri_patterns;
+ */
+
 /**
  * TODO - Fill out remaining properties
```

Comment:
same here

Reply:
Not yet. Going over the TODOs in this file is still to be done.